pkg/doc/did/doc.go

```diff
@@ -201,17 +210,17 @@ type rawDoc struct {
 	Authentication []interface{}            `json:"authentication,omitempty"`
 	Created        *time.Time               `json:"created,omitempty"`
 	Updated        *time.Time               `json:"updated,omitempty"`
-	Proof          *Proof                   `json:"proof,omitempty"`
+	Proof          []interface{}            `json:"proof,omitempty"`
```

Comment:
I would rename it to Proofs

Reply:
Per spec it is proof, just like authentication etc.